### PR TITLE
Add moose submodule to VTB

### PR DESCRIPTION
**Pull request by GiudGiud**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -25,3 +25,6 @@
 [submodule "apps/relap-7"]
 	path = apps/relap-7
 	url = git@github.inl.gov:ncrc/relap-7.git
+[submodule "apps/moose"]
+	path = apps/moose
+	url = git@github.com:idaholab/moose.git
```
Comment (GiudGiud) [inline .gitmodules:30]: this is the correct address right?
is there some sort of firewall rule on civet or something preventing me from adding a submodule ?
@loganharbour

**Comment by loganharbour** [comment]
What is this for?

**Comment by GiudGiud** [comment]
thermochimica for now.
it also makes sense for checking meshing scripts

**Comment by loganharbour** [comment]
Use the https remote. There's no need to use SSH

**Pull request merged**